rust/src/nfs/nfs.rs

```diff
@@ -437,6 +454,104 @@ impl NFSState {
         }
     }
 
+    fn add_rpc_udp_ts_pdu_frame(&mut self, flow: *const Flow, stream_slice: &StreamSlice, input: &[u8], rpc_len: i64) -> Option<Frame> {
+        let rpc_udp_ts_pdu = Frame::new_ts(flow, stream_slice, input, rpc_len, NFSFrameType::RPCPdu as u8);
+        SCLogDebug!("rpc_udp_pdu ts frame {:?}", rpc_udp_ts_pdu);
+        rpc_udp_ts_pdu
+    }
+
+    fn add_rpc_udp_ts_creds(&mut self, flow: *const Flow, stream_slice: &StreamSlice, input: &[u8], rpc_len: i64) {
+        let _rpc_udp_ts_creds = Frame::new_ts(flow, stream_slice, &input[24..], rpc_len - 32, NFSFrameType::RPCCreds as u8);
```

Comment:
can you explain the calculation here?

Reply:
rpc creds is of variable length, so I had to exclude all other fixed-length field to calculate creds.
should I explain this in a comment?

Reply:
we parse the creds, right? So we should know the length?